getting_started/deleting-your-cluster.adoc

```diff
@@ -5,7 +5,7 @@ include::modules/common-attributes.adoc[]
 
 To delete your {product-title} cluster:
 
-. From link:https://cloud.redhat.com/openshift[cloud.redhat.com/openshift], click
+. From link:https://console.redhat.com/openshift[console.redhat.com/openshift], click
  on the cluster you want to delete.
 
 . Click the *Actions* button, then *Delete Cluster*.
```

Comment:
Future content strategy question - the OpenShift Cluster Manager docs also cover deleting clusters, as it's a cluster-level task (not something that happens inside OpenShift - that's how we've been deciding the scope of the OCM docs). Would it be good to link to the OCM docs, or is it OK to have this in two places? https://access.redhat.com/documentation/en-us/openshift_cluster_manager/2021/html-single/managing_clusters/index#archiving-cluster_assembly_cluster-lifecycle